File: code/f4078.pas
program Bai078;
var T, gio, phut, giay: integer;
begin
  readln(T);
  gio := T div 10000;
  phut := (T div 100) mod 100;
  giay := T mod 100;
  write(gio, '  ', phut, '  ', giay);
end.

File: code/f4078.cpp
#include <iostream>
using namespace std;

int main() 
{
    int t,giay,gio,phut;
    cin>>t;
    giay=t%100;
    gio= t/10000;
    phut= (t/100)%100;
    cout<<gio<<" "<<phut<<" "<<giay;
    return 0;
}